firestore-tos-extension/POSTINSTALL.md

```diff
@@ -0,0 +1,154 @@
+### See it in action
+
+You can test out this extension right away!
+
+1. Go to your [Authentication dashboard](https://console.firebase.google.com/project/${param:PROJECT_ID}/authentication/users) in the Firebase console.
+
+1. Click **Add User** to add a test user, then copy the test user's UID to your clipboard.
+
+2. Access the user through any of the authentication methods.
+
+```js
+   // v9
+   import { getAuth, onAuthStateChanged } from "firebase/auth";
+
+   const auth = getAuth();
+   const user = await auth.getUser(uid);
+```
+
+```js
+   // < v9
+   import firebase from "firebase-admin";
+
+   const user = await firebase.auth().getUser(uid);
+```
+
+3. Provide the user / userId into one of the available functions.
+
+### Using the extension
+
+All ackowledgement documents are managed in the Cloud Firestore database.
+
+These can be found in the `${param:COLLECTION_PATH}` collection.
+
+This will also contain two pre-defined subcollections:
+
+- acknowledgements
+- notices
+
+The extension will provide `4` new functions for managing ackowledgement for users.
+
+**Create Notices**
```

Comment:
-We should possibly mention the allowList functionality allows you to specify notices 

-If you need versioning for your notices, we support that out of the box. You can simply create multiple notices with the same notice_type, and the extension will give you a way to query the latest version of the a given notice_type.

Reply:
Updated documentation with the above.